actions/actions.py

```diff
@@ -184,5 +451,23 @@ def run(
         domain: Dict[Text, Any],
     ) -> List[Dict[Text, Any]]:
         age = tracker.get_slot("age")
+        gender = tracker.get_slot("gender")
         province = tracker.get_slot("province")
-        return [AllSlotsReset(), SlotSet("age", age), SlotSet("province", province)]
+        medical_condition = tracker.get_slot("medical_condition")
+        medical_condition_obesity = tracker.get_slot("medical_condition_obesity")
+        medical_condition_diabetes = tracker.get_slot("medical_condition_diabetes")
+        medical_condition_hypertension = tracker.get_slot(
+            "medical_condition_hypertension"
+        )
+        medical_condition_cardio = tracker.get_slot("medical_condition_cardio")
+        return [
+            AllSlotsReset(),
+            SlotSet("age", age),
+            SlotSet("gender", gender),
+            SlotSet("province", province),
+            SlotSet("medical_condition", medical_condition),
+            SlotSet("medical_condition_obesity", medical_condition_obesity),
+            SlotSet("medical_condition_diabetes", medical_condition_diabetes),
+            SlotSet("medical_condition_hypertension", medical_condition_hypertension),
+            SlotSet("medical_condition_cardio", medical_condition_cardio),
```

Comment:
Should we keep this list of slots outside of the class, since it's getting quite big, we could rather have it as a variable, and then use the same list inside of the form, and to generate this list.

Reply:
Not sure I follow what you're suggesting?

Reply:
I'm thinking we should have a variable outside of the class, or we can have one on the class
```
profile_slots = [
    "age",
    "gender",
    "province",
    "medical_condition",
]

profile_conditions = [
    "medical_condition_obesity",
    "medical_condition_diabetes",
    "medical_condition_hypertension",
    "medical_condition_cardio",
]
```
Then here we can do a
```
actions = [AllSlotsReset()]
for slot in HealthCheckProfileForm.profile_slots + HealthCheckProfileForm.profile_conditions:
    actions.append(SlotSet(slot, tracker.get_slot(slot))
return actions
```

Reply:
neat :)